web/src/beta/components/TabButton/index.tsx

```diff
@@ -0,0 +1,49 @@
+import type { FC } from "react";
+
+import { styled } from "@reearth/services/theme";
+
+export type Props = {
+  label: string;
+  onClick?: () => void;
+  selected?: boolean;
+};
+
+const TabButton: FC<Props> = ({ label, onClick, selected }) => {
+  return (
+    <Button onClick={onClick} disabled={selected}>
+      {label}
+    </Button>
+  );
+};
+
+type ButtonProps = {
+  disabled?: boolean;
+};
+
+const Button = styled.button<ButtonProps>`
+  // Button Settings
+  background: ${({ disabled }) => (disabled ? "#232226" : "inherit")};
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px 12px;
+  gap: 10px;
+  height: 35px;
+  left: 0px;
+  top: 0px;
```

Comment:
Can you please make sure all CSS is needed. If you are copy/pasting from Figma I don't think you will get the appropriate CSS always.
For example
- these left and top properties do nothing since this button is not positioned absolutely to anything.
- Probably flex properties aren't needed
- etc

Reply:
Please confirm that we have corrected the points pointed out.